products/pluto/pluto_build/sql/corr_communitydistrict.sql

```diff
@@ -1,6 +1,8 @@
 -- Take the community district value from research table
 INSERT INTO pluto_changes_not_applied
-SELECT DISTINCT b.*
+SELECT DISTINCT
+    b.*,
+    a.cd AS found_value
```

Comment:
Changes/research/corrections are not applied if the "old_value" in the manual corrections csv does not match the found value in pluto. So I've added a column to this QA output to include the actual found value